docs/getting_started.md

```diff
@@ -21,21 +21,15 @@ Generic applications may be run in Shadow. The most important required features
 
```

Comment:
Not related to your change, but I'm not sure what this is trying to rule out. Using signals mabye?

Reply:
I'm not sure either, which is why I just left it. I think much of this file will need to be rewritten.